
INFO:root:load_data. dataset_name = mnist
INFO:root:create_model. model_name = lr, output_dim = 10
INFO:root:/home/cbq/sourcecode/FedML-IoT
E1026 03:14:32.460637200    4435 socket_utils_common_posix.cc:223] check for SO_REUSEPORT: {"created":"@1635189272.460622600","description":"Protocol not available","errno":92,"file":"src/core/lib/iomgr/socket_utils_common_posix.cc","file_line":202,"os_error":"Protocol not available","syscall":"getsockopt(SO_REUSEPORT)"}
INFO:werkzeug: * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
server started. Listening on port 50000
done running
 * Serving Flask app "app" (lazy loading)
 * Environment: production
[31m   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
INFO:werkzeug:127.0.0.1 - - [26/Oct/2021 03:14:39] "[37mPOST /api/register?device_id=0 HTTP/1.1[39m" 200 -
INFO:werkzeug:127.0.0.1 - - [26/Oct/2021 03:14:39] "[37mGET //get-preprocessed-data/0 HTTP/1.1[39m" 200 -
INFO:werkzeug:127.0.0.1 - - [26/Oct/2021 03:14:42] "[37mPOST /api/register?device_id=1 HTTP/1.1[39m" 200 -
INFO:werkzeug:127.0.0.1 - - [26/Oct/2021 03:14:42] "[37mGET //get-preprocessed-data/1 HTTP/1.1[39m" 200 -
INFO:root:client_0 got something from client_1 from ip address 127.0.0.1
INFO:root:add_model. index = 0
INFO:root:b_all_received = False
INFO:root:client_0 got something from client_2 from ip address 127.0.0.1
INFO:root:add_model. index = 1
INFO:root:b_all_received = True
INFO:root:len of self.model_dict[idx] = 2
INFO:root:aggregate time cost: 0
INFO:root:################test_on_server_for_all_clients : 0
indexes of clients: [1, 1]
size = 3
INFO:root:{'training_acc': 0.2411617799688635, 'training_loss': 2.225677122629224}
